pages/404.tsx

```diff
@@ -32,10 +30,11 @@ const FourOhFour: NextPage = () => {
               Sorry, we couldn’t find the page you’re looking for.
             </p>
             <div className="mt-6">
-              <Link href="/">
-                <a className="text-base bg-clip-text text-transparent bg-gradient-to-r from-orange-400 to-pink-600 font-semibold tracking-wide">
-                  Go back home<span aria-hidden="true"> &rarr;</span>
-                </a>
+              <Link
+                className="text-base bg-clip-text text-transparent bg-gradient-to-r from-orange-400 to-pink-600 font-semibold tracking-wide"
+                href="/"
+              >
+                Go back home<span aria-hidden="true"> &rarr;</span>
```

Comment:
"Go Back Home" wording here could be changed to "Return Home" to conciseness to the language interpretation for Screen Readers.